src/main/java/org/octri/clinchit/domain/ClinicianPriority.java

```diff
@@ -0,0 +1,36 @@
+package org.octri.clinchit.domain;
+
+import javax.persistence.Entity;
+
+import org.octri.clinchit.view.Labelled;
+
+/**
+ * The priority as assessed by the clinician
+ * 
+ * <code>
+ * High | Medium | Low
```

Comment:
Do you expect there to be additional priorities? Or would this be better as an enum?

Reply:
There might end up being one or two more, but you're right. Both this and the patient readiness might be better as an enum, since we'll likely be applying css styles to them.

Reply:
Just as an FYI, the boot-generator does support enums for creating the ddl, but I can't remember if I implemented it yet for generating the select options, so you may need to do that manually for now if you want to switch.

Reply:
I'm going to leave this one on the backlog for now since enums aren't yet supported by SelectOption and OptionList. I'd rather have the time to do this right.